src/main/java/me/entity303/openshulker/commands/OpenShulkerCommand.java

```diff
@@ -34,7 +34,7 @@ public boolean onCommand(@NotNull CommandSender commandSender, @NotNull Command
         }
 
         if (args[0].equalsIgnoreCase("reload")) {
-            this._openShulker.reloadConfig();
+            this._openShulker.reloadCfg();
```

Comment:
I don't see a point in creating the reloadCfg method.

Just publicize `InitializeConfig` and execute that.